source/nodejs/adaptivecards-ui-testapp/adaptivecards-ui-testapp.css

```diff
@@ -453,3 +453,62 @@
     border: 1px solid #0078D7;
 }
 
+/*
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+*/
+body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"
+}
```

Comment:
unclear what inserted this.... I'll look in to it...

Reply:
@almedina-ms - it looks like `source/nodejs/adaptivecards-ui-testapp/adaptivecards-ui-testapp.css` is picking up `source/nodejs/adaptivecards-ui-testapp/src/adaptivecards-ui-testapp.css` (note `src`) is this intentional?

Reply:
I guess we don't need this file. Likely it's an file created by plugin by ConcatPlugin, then copied to dist. and it's possible to remove if we change:
```
new ConcatPlugin({
	bundles: [
		{
			dest: 'adaptivecards-ui-testapp.css',
			src: ['./node_modules/adaptivecards-controls/dist/adaptivecards-controls.css', './src/adaptivecards-ui-testapp.css']
		}
	],
}),
new CopyWebpackPlugin({
	patterns: [
	{
		from: 'src/adaptivecards-ui-testapp.css',
		to: '.',
		flatten: true
	},
```
to
```
new ConcatPlugin({
	bundles: [
		{
			dest: './dist/adaptivecards-ui-testapp.css',
			src: ['./node_modules/adaptivecards-controls/dist/adaptivecards-controls.css', './src/adaptivecards-ui-testapp.css']
		}
	],
}),
```